app_flutter/pubspec.yaml

```diff
@@ -35,7 +35,6 @@ dev_dependencies:
     sdk: flutter
   mockito: ^4.0.0
   path: ^1.6.4
-  test: ^1.6.0
```

Comment:
cocoon_service (backend) is dependent on package test. Are dev dependencies ignored?

Reply:
Yup!

Reply:
(assuming that is the dev_dependency of cocoon_service)